dashboard/src/app/projects/list-projects/list-projects.controller.js

```diff
@@ -53,14 +53,20 @@ export class ListProjectsCtrl {
 
     this.profileCreationDate = profilePreferences['che:created'];
 
-    this.dropDownOptionsList = [
+    let ctrl = this;
```

Comment:
I would say that we no longer need hack "var = this" if we use arrow function instead of function () in order to use